MobileCenterCrashes/MobileCenterCrashes/Internals/MSCrashesPrivate.h

```diff
@@ -70,10 +70,7 @@ typedef struct MSCrashesCallbacks {
   MSCrashesPostCrashSignalCallback handleSignal;
 } MSCrashesCallbacks;
 
-// TODO: Mach exception handler is not supported on tvOS.
-#if !TARGET_OS_TV
```

Comment:
I still don't know this is a right thing to have this property on tvOS. If it is not supported by tvOS, better to remove instead of keeping it. However if you are fine with change, I'm also okay.

Reply:
I am fine as soon as it's not public.